vault/plugin_catalog_test.go

```diff
@@ -129,6 +130,8 @@ func TestPluginCatalog_List(t *testing.T) {
 	}
 
 	if len(plugins) != len(builtinKeys) {
+		log.Printf("%s", builtinKeys)
```

Comment:
Looks like debug logs. If so, can we remove these?

Reply:
Removed